consensus/dummy/consensus.go

```diff
@@ -121,7 +122,7 @@ func (eng *DummyEngine) verifyCoinbase(header *types.Header, parent *types.Heade
 	// we fetch the configured coinbase at the parent's state
 	// to check against the coinbase in [header].
 	if configuredAddressAtParent != header.Coinbase {
-		return fmt.Errorf("%w: %v does not match required coinbase address %v", vmerrs.ErrInvalidCoinbase, header.Coinbase, configuredAddressAtParent)
+		return fmt.Errorf("%w: %v does not match required coinbase address %v", customvm.ErrInvalidCoinbase, header.Coinbase, configuredAddressAtParent)
```

Comment:
Note the 3 custom vm errors were moved to `core/vm/errors_ext.go`, let me know if you feel they should be in another location.

Reply:
I slightly prefer moving all custom stuff under `plugin/evm`

Reply:
Does the following work?

- `ErrSenderAddressNotAllowListed` is used in `core` and `core/txpool`: move to plugin/evm/core
- `ErrInvalidCoinbase` is used in `consensus/dummy`: move to plugin/evm/consensus
- `ErrAddrProhibited` is unused: let's remove it

Reply:
🔼  done that